graylog2-web-interface/src/components/lookup-tables/DataAdapterTableEntry.jsx

```diff
@@ -22,10 +29,18 @@ const DataAdapterTableEntry = React.createClass({
   },
 
   render() {
+    const errorOverlay = !this.props.error ? null : (
+      <Popover id="popover-table-error" title="Lookup Table problem" style={{ maxWidth: 400 }}>
```

Comment:
These `Popovers` and `OverlayTrigger` are basically the same as the ones in `LUTTableEntry`. It would help extracting them into a component that could read an error state and render it.
I see this issue more as an improvement than a big issue, but I wanted to point it out anyway.